js/counting.js

```diff
@@ -0,0 +1,34 @@
+export function counting() {
```

Comment:
i think it is quite the same content as in Create.js, if this is like that you wouldn t need this anymore

Reply:
okay, thank you :)